src/app/home/home.component.html

```diff
@@ -17,6 +17,23 @@
       </mat-card-title>
     </mat-card-header>
 
+    <mat-card-content>
+        
+      <div fxLayout="column">
+
+        <mat-form-field>
```

Comment:
refer to accounting section, make autocomplete HTML as well as code similar to auto completes being used in accounting section (for eg: gl account autocomplete)

Reply:
I have used mat-autocomplete. What's the error in here?

Reply:
There are multiple ways of implementing autocomplete, make it consistent with rest of app.

Reply:
Are there any autocomplete components similar to this? I found only filtering techniques which were used to filter the mat-table.

Reply:
use your text editor to search for gl account autocomplete in code base, or just search for autocomplete, which code editor do you use?

Reply:
VS Code

Reply:
> use your text editor to search for gl account autocomplete in code base, or just search for autocomplete, which code editor do you use?

I searched this. But no result found

Reply:
Found in closing-entries. I'll fix. Thanks :)

Reply:
Done. Please review